src/Administration/Resources/app/administration/src/module/sw-customer/component/sw-customer-imitate-customer-modal/index.js

```diff
@@ -63,7 +63,12 @@ export default {
         salesChannelDomainCriteria() {
             const criteria = new Criteria();
             criteria.addAssociation('salesChannel');
-            criteria.addFilter(Criteria.equals('salesChannel.typeId', Shopware.Defaults.storefrontSalesChannelTypeId));
+            criteria.addFilter(
+                Criteria.multi('AND', [
+                    Criteria.equals('salesChannel.typeId', Shopware.Defaults.storefrontSalesChannelTypeId),
+                    Criteria.equals('salesChannel.active', true),
+                ]),
+            );
```

Comment:
We don't need the multi-filter here.
```suggestion
            criteria.addFilter(Criteria.equals('salesChannel.typeId', Shopware.Defaults.storefrontSalesChannelTypeId));
            criteria.addFilter(Criteria.equals('salesChannel.active', true));
```